samples/precompiled/Monitor.cs

```diff
@@ -20,26 +40,54 @@ public static class Monitor
         public static async Task Run([OrchestrationTrigger] DurableOrchestrationContext monitorContext, TraceWriter log)
         {
             MonitorRequest input = monitorContext.GetInput<MonitorRequest>();
+            if (!monitorContext.IsReplaying)
```

Comment:
While I normally advise against this, I will make an exception here: the code would be more readable if we made all these conditional log statements a one-liner: 
```csharp
if (!monitorContext.IsReplaying) log.Info(...);
```